src/tabs/tab.directive.ts

```diff
@@ -29,21 +29,15 @@ export class TabDirective implements OnInit, OnDestroy {
   }
 
   set customClass(customClass: string) {
-    if (this._customClass && this._customClass !== customClass) {
-      this.renderer.removeClass(
-        this.elementRef.nativeElement,
-        this._customClass
-      );
-    }
-
     this._customClass = customClass;
+    this._customClass.split(' ').forEach((element: string) => this.setElementClass = element);
+  }
 
-    if (this._customClass) {
-      this.renderer.addClass(
-        this.elementRef.nativeElement,
-        this._customClass
-      );
-    }
+  set setElementClass(customClass: string) {
+    this.renderer.addClass(
```

Comment:
what if list of classes changes? for example, if user pass `class-1 class-2` and then `class-2 class-3`. in this case all 3 classes will be appended